plugins/chat/app/serializers/chat/block_serializer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Chat
+  class BlockSerializer < ApplicationSerializer
+    attributes :type, :elements
+
+    def type
+      object["type"]
+    end
+
+    def elements
+      object["elements"].map do |element|
+        serializer = Chat::BlockSerializer.element_serializer_for(element["type"])
```

Comment:
We can use `self.class` instead of repeating the module/class name.

Reply:
I will change it, but my argument for doing this is grepability

Reply:
To see where that class in particular is used? If so, by using `self.class`, it’s not necessary anymore, as it will only refer to that class. And it also means you’re already in said class. Another pro is that it’s less brittle, as if the file is moved around, or renamed or whatever, that code doesn’t need to be changed :grin:

Reply:
if signature changes?